src/test/java/teammates/test/cases/logic/CoursesLogicTest.java

```diff
@@ -438,12 +438,24 @@ private void testGetCoursesForStudentAccount() throws Exception {
         assertEquals(2, courseList.size());
 
         CourseAttributes course1 = dataBundle.courses.get("typicalCourse1");
-        assertEquals(course1.getId(), courseList.get(0).getId());
-        assertEquals(course1.getName(), courseList.get(0).getName());
+        int temp = 0;
```

Comment:
`temp` is not a good variable name, and not suggestive of what it is about. Something like `boolean isCourse1Present = false;` which will be set to `true` later is better.
That way you also do not need to loop twice; simply keep two boolean checkers.